### PR TITLE
Display customers that haven't placed an order

### PR DESCRIPTION
**Pull request by sydneywait**
# Description
I have added a query parameter ?active="false" to display any customers that have not yet placed an order.  It also works with the additional query parameter to search by first or last name (q="{SearchParam}")

## Related Ticket(s)

[11](https://trello.com/c/jR6kH4TF/11-ticket-11-display-customers-that-havent-placed-an-order)
Previous ticket--
[1](https://trello.com/c/Mm5h5VEl/1-ticket-1-allow-developers-to-access-the-customer-resource) Accessing the customer resource

## Steps to Test
You will FIRST need to add a customer to the database that is not associated with any orders.  You can do this in SSMS or the sql query window in Visual Studio using the command:
INSERT INTO Customer (FirstName, LastName) VALUES ('Elon', 'Musk')

Or in Postman using a "Post" method to https://localhost:5001/api/customer and the body:
{
"FirstName": "Elon",
"LastName": "Musk"
}
NOTE: you need to compile and run any working version of the program to post the customer using postman (including this test version).

Make sure you save, add, and commit on your own branch before checking out to master and then to
````sw-customer-no-order```` (use git fetch --all if you can't already access it)

Perform a ````git pull origin sw-customer-no-order```` to get the latest version.

type ````start BangazonAPI.sln```` if you don't already have it open

In the Solution Explorer, right click on 'TestBangazonApi' and select 'Run Tests'.   They should all pass.

Compile and run the program, and in the browser type
````https://localhost:5001/api/customer```
to see all the customers

Type:
````https://localhost:5001/api/customer?active=false````

to see Elon Musk and any other customer not associated with an order. 

````https://localhost:5001/api/customer?active=false&q=Elo````
Should still just show you Elon Musk.  If you type letters that are not in Elon's name or add another customer not associated with an order it will show the correct results accordingly (no customer, a different customer, etc)

# Checklist:
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Cluttered04** [comment]
🐘

File: BangazonAPI/Controllers/CustomerController.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public SqlConnection Connection
             }
 
             [HttpGet]
-            public async Task<IActionResult> Get(string include, string q)
+            public async Task<IActionResult> Get(string include, string q, string active)
```
Comment (jordan-castelloe) [inline BangazonAPI/Controllers/CustomerController.cs:33]: Let's change `active` to a boolean rather than a string!

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
One tiny change in data type, but otherwise works as described and looks great!

File: BangazonAPI/Controllers/CustomerController.cs (hunk 1 above)
Comment (sydneywait) [reply BangazonAPI/Controllers/CustomerController.cs:33]: I had it as a boolean and it was always resulting in active==false being true?  Which is why I changed it to be a string.  Maybe I can change it to that and show you what I mean.

**Pull request merged**